### PR TITLE
Feat/put products

### PR DESCRIPTION
**Pull request by devBK84**

### CODE REVIEW

File: backend/src/main/java/com/github/devbk84/backend/controller/ProductController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    @GetMapping
+    public List<Product> getAllProducts(){
+
+        return productService.getAllProducts();
+    }
+
+    @GetMapping(path = "/{id}")
+    public Product getProductsById(@PathVariable String id){
+        return productService.getProductsById(id);
```
Comment (MoinMarcell) [inline backend/src/main/java/com/github/devbk84/backend/controller/ProductController.java:28]: You just recive one product, you should rename the methods to getProductById 
```suggestion
    public Product getProductById(@PathVariable String id){
        return productService.getProductById(id);
```

File: backend/src/main/java/com/github/devbk84/backend/service/ProductService.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    public ProductService(ProductRepo productRepo) {
+        this.productRepo = productRepo;
+    }
+
+    public List<Product> getAllProducts() {
+        return productRepo.findAll();
+    }
+
+    public Product getProductsById(String id) {
```
Comment (MoinMarcell) [inline backend/src/main/java/com/github/devbk84/backend/service/ProductService.java:23]: As mentioned above, you get one produtc, not many :)
```suggestion
    public Product getProductById(String id) {
```

File: backend/src/test/java/com/github/devbk84/backend/controller/ProductControllerTest.java [hunk 3]
```diff
@@ -39,4 +45,15 @@ void getAllProducts() throws Exception {
                 .andExpect(content().json("[]"));
     }
 
+    @Test
+    @DirtiesContext
+    void getProductsById() throws Exception {
```
Comment (MoinMarcell) [inline backend/src/test/java/com/github/devbk84/backend/controller/ProductControllerTest.java:50]:
```suggestion
    void getProductById() throws Exception {
```

File: backend/src/test/java/com/github/devbk84/backend/controller/ProductControllerTest.java [hunk 4]
```diff
@@ -39,4 +45,15 @@ void getAllProducts() throws Exception {
...
+        productRepo.save(new Product("69",
+                "Milch", "Test",
+                "orderFav", new BigDecimal(0)));
+        mockMvc.perform(get("/api/products/69"))
+                .andExpect(status().isOk());
```
Comment (MoinMarcell) [inline backend/src/test/java/com/github/devbk84/backend/controller/ProductControllerTest.java:55]: For later it is easier to seperate the product in a own Variable, so you can easily modify the test later :)
```suggestion
        Product product = new Product("69",
                "Milch", "Test",
                "orderFav", new BigDecimal(0));
        productRepo.save(product);
        mockMvc.perform(get("/api/products/" + product.id()))
                .andExpect(status().isOk());
```

**Review by MoinMarcell — CHANGES_REQUESTED** [review]
LGTM, I mentioned some changes and check your language. You mix up english and german, choose one language :) And also check you codesmell in the sonar comment for backend

**Review by fannie-id — APPROVED** [review]

**Review by fannie-id — APPROVED** [review]

**Pull request merged**